guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -173,3 +173,22 @@ endif::[]
 . Click *Generate*.
 . Copy the generated `curl` command.
 . On the host that you want to register, run the `curl` command as `root`.
+
+.CLI procedure
+
+. Use the `hammer host-registration generate-command` to generate the `curl` command to register the host.
+. On the host that you want to register, run the `curl` command as `root`.
+
+For more information, enter the `hammer host-registration generate-command --help` command.
+
+.Ansible procedure
+
+* Use the `{ansible-namespace}.registration_command` module.
+
+For more information, enter the `ansible-doc {ansible-namespace}.registration_command` command.
+
+.API procedure
+
+* Use the `POST /api/registration_commands` resource.
+
+For more information, see the full API reference at `\https://{foreman-example-com}/apidoc/v2.html`.
```

Comment:
You could also link directly to the registration command help, if you wanted to:
```suggestion
For more information, see the API reference at `\https://{foreman-example-com}/apidoc/v2/registration_commands/create.html`.
```

Reply:
I think I'd prefer to stick with the generic link. If there is a procedure that would require us to list multiple API resources, we wouldn't want to have a link for each one.